data/mods/MindOverMatter/items/misc.json

```
@@ -18,5 +18,42 @@
       "description": "Pick a card.  Any card.",
       "effect_on_conditions": [ "EOC_ZENER_DECK" ]
     }
+  },
+  {
+    "id": "force_field_generator",
+    "type": "TOOL",
+    "category": "other",
+    "name": { "str": "device labeled OS647-P", "str_pl": "devices labeled OS647-P" },
+    "description": "A small plastic device about the size of two smartphones stacked on top of each other, with a toggle on one end and \"OS647-P\" written prominently on one side above the XEDRA logo.  There is an intention on the other side the right size to contain a matrix crystal.",
```

```suggestion
    "description": "A small plastic device about the size of two smartphones stacked on top of each other, with a toggle on one end and \"OS647-P\" written prominently on one side above the XEDRA logo.  There is an indention on the other side the right size to contain a matrix crystal.",
```
I think there is a typo there.

---

You're right.